recipes/r-rcppplanc/conda_build_config.yaml

```diff
@@ -0,0 +1,14 @@
+# Please consult conda-forge/core before doing this
+c_stdlib_version:
+  - "11"           # [osx and x86_64]
+  - "2.17"         # [linux and x86_64]
+MACOSX_SDK_VERSION:        # [osx and x86_64]
+  - "11.3"                # [osx and x86_64]
```

Comment:
Can you say why this is needed?

Reply:
Its CRAN's minimum SDK target and I'm fairly sure (although I may be misremembering, it was a few months ago) I was getting build issues with conda-forge's default.

Reply:
Can you remove it and corresponding `c_stdlib_version` entry as well to see if there are errors?

Reply:
On it. Was trying to figure out why I was still getting 6 builds per os and realized I left a build-string template in there.

Reply:
https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=956616&view=logs&j=e35d4f76-8ff2-5536-d795-df91e63eb9f7&t=fa7b4b17-b6ff-5c9c-8cfc-15f888c92310&l=1641

well, there's your answer.

Reply:
Didn't know about the D_LIBCPP_DISABLE_AVAILABILITY option-I think it works now?

Reply:
Yup.